tests/unit/amo/components/TestFeaturedCollectionCard.js

```diff
@@ -55,6 +55,20 @@ describe(__filename, () => {
     expect(root.find(LandingAddonsCard)).toHaveProp('className', className);
   });
 
+  it('renders a theme class name to LandingAddonsCard if the collection contains theme addons', () => {
+    const root = render({ isTheme: true });
+    expect(root.find(LandingAddonsCard)).toHaveClassName(
+      'FeaturedCollection--theme',
+    );
+  });
+
+  it('does not render a theme class name to LandingAddonsCard if the collection contains extension addons', () => {
```

Comment:
Because the logic isn't actually tied to "if the collection contains theme addons", but rather to the value of the isTheme property, the test should be worded that way as well.

Reply:
I am still not sure about using isTheme prop :/ but I will try it out

Reply:
Are you still unsure about this? It seems to work pretty well to me. What issues do you have with it?